htdocs/comm/propal/card.php

```diff
@@ -966,7 +966,7 @@
 				$tmpprodvat = price2num(preg_replace('/\s*\(.*\)/', '', $prod->tva_tx));
 
 				// if price ht is forced (ie: calculated by margin rate and cost price). TODO Why this ?
-				if (!empty($price_ht)) {
```

Comment:
Don't you think we should replace here with
if (!empty($price_ht) || $price_ht === '0') {
Because if price_ht is empty string '' for example, we must no enter the test, and we must of it is '0'